### PR TITLE
Zip code lookups

### PR DESCRIPTION
**Pull request by JoeBartelmo**

### CODE REVIEW

**Review by Dalton098 — APPROVED** [review]
wowie great work ty ty ty ty ty ty

File: backend/server.ts [hunk 1]
```diff
@@ -13,6 +14,16 @@ const angularPath = (__dirname.endsWith("dist") ? '/../../' : '/../') + 'fronten
 
 app.use('/weather', proxy('https://api.weather.gov'));
 app.use('/mesonet', proxy('https://mesonet.agron.iastate.edu'));
+app.get('/zipcode/:zipCode', (req, res) => {
+  const zip = req.params.zipCode;
+  console.log('Performing lookup for ' + zip);
+  const result = typeErasedCodes[`${zip}`];
+  if (result != undefined) {
+    console.log('found', result);
```
Comment (Joe102192) [inline backend/server.ts:22]: wowza bonanza this is big

**Review by Joe102192 — APPROVED** [review]

File: backend/server.ts (hunk 1 above)
Comment (JoeBartelmo) [reply backend/server.ts:22]: BIF

**Comment by JoeBartelmo** [comment]
> wowie great work ty ty ty ty ty ty

I can only behold

**Pull request merged**